includes/payments/WC_WooMercadoPago_TicketGateway.php

```diff
@@ -365,6 +368,7 @@ public function payment_fields()
         $amount = $amount - $discount + $comission;
 
         $logged_user_email = (wp_get_current_user()->ID != 0) ? wp_get_current_user()->user_email : null;
+        $discount_action_url = get_site_url() . '/index.php/woocommerce-mercadopago/?wc-api=WC_WooMercadoPago_TicketGateway';
```

Comment:
Não poderia usar a classe do Abstract?
$this->discount_action_url = get_site_url() . '/index.php/woocommerce-mercadopago/?wc-api=' . get_class($this);